modules/apps/foundation/users-admin/users-admin-web/src/main/resources/META-INF/resources/view_flat_users.jsp

```diff
@@ -266,7 +266,13 @@ boolean showRestoreButton = (searchTerms.getStatus() != WorkflowConstants.STATUS
 		}
 		else {
 			if (filterManageableOrganizations && !UserPermissionUtil.contains(permissionChecker, ResourceConstants.PRIMKEY_DNE, ActionKeys.VIEW)) {
-				userParams.put("usersOrgs", ArrayUtil.toLongArray(user.getOrganizationIds()));
+				long[] organizationIds = user.getOrganizationIds();
+
+				if (ArrayUtil.isEmpty(organizationIds)) {
+					organizationIds = new long[] {0};
```

Comment:
Hey Pei-Jung,

Why is this needed? Is it because organizationIds is null? 
If so, shouldn't we change the implementation of user.getOrganizationIds() instead so that such a thing never happens.